### ISSUE TITLE
When removing a check, sensu service not refresh

### ISSUE THREAD

**Issue by jothoma1** [sensu/sensu-puppet#782]
## Description of problem

- What did you do?
I'm trying to remove a check with

> ensure => absent
- What happened?
It works but it did not refresh the service sensu-client
- What did you expect to happen?
Refresh the service in order to have the good configuration.
- How can someone reproduce the problem?
1) Create a check
2) try to remove it with ensure => absent
3) sensu has not refresh so the check is still present

## Command used and debugging output

- What does your puppet config look like (including any hiera config)
- Is this a masterless or master based puppet setup?

## Platform and version information

- Your OS:  CentOS 7
- Your Ruby version: 2.4
- Your version of Puppet:  3.8.7
- Your version of Sensu:  1.0
- Your version of this module:

**Comment by alvagante** [comment]
@jothoma1 https://github.com/sensu/sensu-puppet/pull/785 should address this, can you confirm that?

**Comment by alvagante** [comment]
@ghoneycutt I can confirm that this issue is now fixed in current master, so I suppose it can be closed
```
==> ubuntu1604-client: Notice: /Stage[main]/Main/Sensu::Check[check_cpu]/Sensu::Write_json[/etc/sensu/conf.d/checks/check_cpu.json]/File[/etc/sensu/conf.d/checks/check_cpu.json]/ensure: removed
==> ubuntu1604-client: Notice: /Stage[main]/Sensu::Client/Service[sensu-client]: Triggered 'refresh' from 1 events

==> el7-client: Notice: /Stage[main]/Main/Sensu::Check[check_cpu]/Sensu::Write_json[/etc/sensu/conf.d/checks/check_cpu.json]/File[/etc/sensu/conf.d/checks/check_cpu.json]/ensure: removed
==> el7-client: Notice: /Stage[main]/Sensu::Client/Service[sensu-client]: Triggered 'refresh' from 1 events
```

**Comment by ghoneycutt** [comment]
Released in v2.50.0
